frontend/guide_sidebar.py

```diff
@@ -373,5 +373,88 @@ def fillSideBar(county=None, state=None, CAA_value=0, CWA_value=0, RCRA_value=0)
 if st.session_state.button:
     # User opens the sidebar
     with st.sidebar:
-        fillSideBar() #starter
-        # fillSideBar("Niagara County", "New York", 72, 73, 74)
+        # fillSideBar() #starter
+        fillSideBar("Niagara County", "New York", 72, 73, 74)
+
+
+# Connect file to style.css
+# FIX PATH AS NEEDED
+with open("C:\\Users\\owenc\\Hack4Impact\\EDGI_Website\\css\\style.css") as f:
+    st.markdown(
+        f"""<style>{f.read()}</style> <body>
+
+  </body>""",
+        unsafe_allow_html=True,
+    )
+
+# Create sections for anchor tags and background info
+col1, col2 = st.columns([0.4, 0.6])
+
+with col1:
```

Comment:
it would be cleaner to refactor the code for the anchor tags/info about stats into separate file, sorry if that wasn't clear in the sprint spec